releasenotes/notes/36912.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
```

Comment:
I don't think we need a bug fix release notes for a not yet released feature?

Reply:
Sorry I actually meant that we don't need a release note for this change at all. You can simply add `release-notes-none` label to the PR.